drools-game-engine-services/src/main/java/org/drools/game/services/endpoint/impl/GameServiceImpl.java

```diff
@@ -21,28 +21,39 @@
 import java.util.List;
 import java.util.Map;
 import java.util.UUID;
+
 import javax.enterprise.context.ApplicationScoped;
 import javax.enterprise.inject.Instance;
 import javax.inject.Inject;
+
+import org.drools.game.capture.flag.model.Zone;
 import org.drools.game.core.BaseGameConfigurationImpl;
 import org.drools.game.core.BasePlayerConfigurationImpl;
+import org.drools.game.core.api.Command;
+import org.drools.game.core.api.Context;
 import org.drools.game.core.api.GameSession;
+import org.drools.game.core.api.StoreFacts;
 import org.drools.game.model.api.Player;
 import org.drools.game.services.endpoint.api.GameService;
 import org.drools.game.services.infos.GameSessionInfo;
+import org.kie.api.runtime.KieSession;
+import org.kie.api.runtime.ObjectFilter;
 
 @ApplicationScoped
 public class GameServiceImpl implements GameService {
 
     @Inject
     private Instance<GameSession> sessions;
-
+    
+    @Inject
+    private Instance<StoreFacts> storeFacts;
```

Comment:
We might need to load this from a configuration file, it could be a JSON file that describes the initial set of facts that needs to be loaded for the game. It doesn't make too much sense to expect to get this from a remote client. So I totally agree, here we need a configuration interface to be able to load those configurations from a json file. I will create an issue for you about that. If that's ok with you.